config/default/kustomization.yaml

```diff
@@ -14,4 +14,6 @@ secretGenerator:
 patchesStrategicMerge:
 - manager_creds_patch.yaml
 resources:
+- ../crd
 - ../base
+- ../bindings/clusterspaced
```

Comment:
I think in future we should refactor `credsbased/clusterscope`, `credsbased/namespacescope`, `rolebased/clusterscope`, and `rolebased/namespacescope` to depend on two common installers, `clusterscope` and `namespacescope`. They would then patch in their specific authentication mechanisms. This will reduce duplication of dependencies and make edits easier since they have to be done in fewer places.

I'm not necessarily saying we should do this now, just trying to share the little knowledge of Kustomize I have.